wikitongues/wikitongues/config/load_configs.py

```diff
@@ -70,7 +70,7 @@ def load_item_airtable_datastores(config):
             config_item_table['table_name'], config_item_table['id_column'],
             config_item_table['page_size'], OffsetUtility.read_offset(),
             config_item_table['max_records']),
-        config_item_table.getboolean('fake'))
+        eval(config_item_table['fake']))
```

Comment:
`bool(config_item_table['fake'])` is a more robust way to get the bool value:
```
>>> bool('True')
True
>>> bool('true')
True
>>> bool()
False
>>> bool(None)
False
>>> bool('')
False
```

Reply:
bool('false') and bool('False') will both return True and since the fake property will always be either be True or False in the config file and that's why I used it. If it is not save I can try to find a different way to implement it.

Reply:
Ah, good catch. Let's stick with `eval` then, but let's make it case-insensitive, and default to `False` if the `fake` property is missing from the config